pkg/commands/artifact/io.go

```diff
@@ -0,0 +1,47 @@
+package artifact
```

Comment:
I think we should move this logic here.
https://github.com/aquasecurity/trivy/blob/main/pkg/misconf/scanner.go